CMakeLists.txt

```diff
@@ -74,22 +74,23 @@ set( EIGEN_INC ${EXTERNAL_DIR}/Eigen )
 set( ASSIMP_INC ${EXTERNAL_DIR}/Assimp/include )
 if( APPLE )
     if ( ${CMAKE_BUILD_TYPE} STREQUAL "Debug" )
-        set( ASSIMP_LIB "${CMAKE_CURRENT_SOURCE_DIR}/lib/libassimpd.dylib" )
+        set( ASSIMP_LIB "${CMAKE_SOURCE_DIR}/lib/libassimpd.dylib" )
```

Comment:
Leave CMAKE_CURRENT_SOURCE_DIR, so that people including Radium in a larger CMake project won't be sad :)